Modules/GridView/GridView.swift

```diff
@@ -0,0 +1,97 @@
+import Foundation
+import JamitFoundation
+import UIKit
+
+// TODO: Write documentation for public interface!
+
+public final class GridView<ItemView: StatefulViewProtocol>: StatefulView<GridViewModel<ItemView.Model>>,
+    UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+    private class ItemViewCell: ContainerCollectionViewCell<ItemView> {}
+
+    private lazy var collectionViewLayout: UICollectionViewLayout = UICollectionViewFlowLayout()
+    private lazy var collectionView: UICollectionView = .init(frame: .zero, collectionViewLayout: collectionViewLayout)
+
+    public var isScrollEnabled: Bool {
+        get { collectionView.isScrollEnabled }
+        set { collectionView.isScrollEnabled = newValue }
+    }
+
+    public override var intrinsicContentSize: CGSize {
+        return collectionView.contentSize
+    }
+
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+
+        collectionView.backgroundColor = .clear
+        addSubview(collectionView)
+        collectionView.clipsToBounds = false
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(collectionView)
```

Comment:
Why are you adding the collectionView multiple times as subview? I guess one call is superfluous.

Reply:
I think that was a mistake after making a git rebase :)